seed/csharp-sdk/file-upload/src/SeedFileUpload/Service/ServiceClient.cs

```diff
@@ -20,22 +20,40 @@ internal ServiceClient(RawClient client)
         CancellationToken cancellationToken = default
     )
     {
+        var multipartForm = new RawClient.MultipartFormRequest
+        {
+            BaseUrl = _client.Options.BaseUrl,
+            Method = HttpMethod.Post,
+            Path = "",
+            Options = options,
+        };
+        multipartForm.AddStringPart("maybe_string", request.MaybeString);
+        multipartForm.AddStringPart("integer", request.Integer);
+        multipartForm.AddFileParameterPart("file", request.File);
+        multipartForm.AddFileParameterParts("file_list", request.FileList);
+        multipartForm.AddFileParameterPart("maybe_file", request.File);
+        multipartForm.AddFileParameterParts("maybe_file_list", request.MaybeFileList);
+        multipartForm.AddStringPart("maybe_integer", request.MaybeInteger);
+        multipartForm.AddStringParts("optional_list_of_strings", request.OptionalListOfStrings);
+        multipartForm.AddJsonParts("list_of_objects", request.ListOfObjects);
+        multipartForm.AddJsonPart("optional_metadata", request.OptionalMetadata);
+        multipartForm.AddJsonPart("optional_object_type", request.OptionalMetadata);
+        multipartForm.AddStringPart("optional_id", request.OptionalId);
+        multipartForm.AddJsonParts("list_of_objects_with_optionals", request.ListOfObjectsWithOptionals);
+        multipartForm.AddJsonPart("alias_object", request.AliasObject);
+        multipartForm.AddJsonParts("list_of_alias_object", request.ListOfAliasObject);
+        multipartForm.AddJsonParts("alias_list_of_object", request.AliasListOfObject);
```

Comment:
Discussed offline - this is purely cosmetic so it's fine to start this way, but we might eventually refactor this so that the `null` checks and the `foreach` statements are more explicit in the generated method.